src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -11,9 +11,9 @@ public abstract class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the entity identified by the user. \n"
-            + "Parameters: entity (person or appt) \n "
-            + "Example: " + COMMAND_WORD + "appt \n";
+            + ": Deletes all entities of a certain type. \n"
+            + "Parameters: ENTITY_TYPE (person/appt)\n "
+            + "Example: " + COMMAND_WORD + " appt \n";
```

Comment:
Should we use `ParserUtil.APPOINTMENT_ENTITY_STRING` instead of writing `appt` manually?

Reply:
OK, I'll make the refactor for this.